lametro/templates/lametro/board_members.html

```diff
@@ -19,6 +19,13 @@
 
 {% cache 86400 members_wrapper 'members' %}
 
+<div class="container-fluid">
+    <br>
+    <div class="alert alert-danger mt-3" role="alert">
+        On December 15, 2021, the County of Los Angles Citizens Redistricting Commission passed resolutions to adopt a final redistricting <a href='https://redistricting.lacounty.gov/wp-content/uploads/2021/12/LA-County-CRC-Resolution-Adopting-Redistricting-Plan.pdf'>plan</a> and <a href='https://redistricting.lacounty.gov/wp-content/uploads/2021/12/LA-County-CRC-Resolution-Adopting-Redistricting-Report.pdf'>report</a>. Click <a href='https://redistricting-lacounty.hub.arcgis.com/'>here</a> to view the final redistricting plan and all other submitted maps in the Redistricting Hub. The map on this page will be updated once the County provides finalized maps.
```

Comment:
These links are a bit hard to read in the red. If they're still a bit subdued with the change in color scheme, how about bolding them?